docs/deployment/README.md

```diff
@@ -342,32 +327,7 @@ elasticsearch_security_group_id = sg-12345
 
 Your data persistence resources are now deployed.
 
-### Deploy the Cumulus Message Adapter layer (DEPRECATED)
```

Comment:
For existing cumulus instance, this information is still useful if DAAC wants to deploy the layer manually or use the module from `cumulus-template-deploy` repo.

Reply:
@jennyhliu I conferred with Matt since these "deprecated" sections are in past versions when they do a search the DAAC user will be able to find this information and so we can remove this for the newly published version. If this information is still helpful in the existing instance, we might need an updated context.